src/test/visit_testing

```diff
@@ -0,0 +1 @@
+py_src
```

Comment:
Can you provide the motivation for this because I don't understand why it was needed.

Reply:
Third paragraph of opening issue comments.

Reply:
I read that, but still don't understand.  What were you attempting that didn't work without this?
I run the test suite in its entirety, or single tests regularly while doing development and haven't run into an issue.

Reply:
Its related to this email I sent over weekend...

```
Hi All,
 
We apparently now run visit testing using imports from the build’s lib dir looking for “visit_testing”
 
This means when you edit VisIt’s test harness python code in src, it has no effect on your testing.
 
There is no adjustment to PYTHONPATH env variable that can have that effect because there is no “visit_testing” in src.
 
Adding a symlink named `visit_testing` to src/test to `py_src` and adjusting PYTHONPATH in run_visit_test_suite.sh does fix but then defeats other modality I think.
 
But, it would be good to have an easy way to run testing and edit the testing code that is being run.
```

Bottom line, you cannot do any development on the tools in `src/test/py_src` without it. Thats because those tools get copied to a `site-packages` file in the build dir and then depend on being found via an `import visit_testing` directive and thereafter only run from there. This kept me guessing for more than an hour over the weekend. The symlink makes it possible for that `import visit_testing` to still work while pointing at the actual source files, not their copies in the build dir.

Reply:
Doesn't that just mean you need to rebuild after changing anything in src/test/py_src ?

Reply:
Oh, well...I've never had to do that before. I could change `.py` code and run it. I don't recall ever having to `make` to run changes to `.py` code. I see it as kinda sorta already made.

If that is the new practice now, its news to me. I mean, I can learn to operate that way but I haven't up until now and the thought never occured to me that `make` is the right answer here.

I just tried `make` in `build/test` and it has the desired effect. I wish I had known this over the weekend 😢 

I can remove the symlink but I think I'll still keep a comment in `run_visit_test_suite.sh.in` to remind developers to do that.

Thanks for the explanation.